src/common/arrstr.cpp

```diff
@@ -573,3 +579,162 @@ wxArrayString wxSplit(const wxString& str, const wxChar sep, const wxChar escape
 
     return ret;
 }
+
+
+namespace
+{
+    enum wxStringFragmentType
+    {
+        wxFRAGMENT_TYPE_EMPTY = 0,
+        wxFRAGMENT_TYPE_ALPHA = 1,
+        wxFRAGMENT_TYPE_DIGIT = 2
+    };
+
+
+    // ----------------------------------------------------------------------------
+    // wxStringFragment
+    // ----------------------------------------------------------------------------
+    // 
+    // Lightweight object returned by wxString::GetNaturalFragment()
```

Comment:
I hate to nag you about the same comment for the third time but perhaps ` wxString::GetNaturalFragment()` should be be replaced with ` GetFragment().`?

Reply:
I'm sorry, it's because I'm being an idiot. oh god, another full build test because of a comment change.